csodiaq/identifier/matchingFunctions.py

```diff
@@ -1,11 +1,95 @@
 import pandas as pd
+import numpy as np
+from numba import njit
+from enum import Enum
 
+
+class Increment(Enum):
+    NEITHER = 0
+    LIBRARY = 1
+    QUERY = 2
```

Comment:
We talked about this, and I've determined the function name should be changed (it doesn't technically "increment") but otherwise this will stay the same.